dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/HdfsPermissionExtractionTask.java

```diff
@@ -41,19 +44,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class HdfsPermissionExtractionTask
-    implements Task<Void>, HdfsPermissionExtractionDumpFormat {
+public class HdfsPermissionExtractionTask extends AbstractTask<Void>
+    implements PermissionExtraction, HdfsPermissionExtractionDumpFormat {
```

Comment:
I think that implementing both `PermissionExtraction` and `HdfsPermissionExtractionDumpFormat` is too much. I would suggest removing one of them and then adjusting:
- the references, e.g. `ZIP_ENTRY_NAME` in line 57 vs. `PermissionExtraction.ZIP_ENTRY_NAME` in line 72,
- imports.